### ISSUE TITLE
converting to a new timezone and back doesn't give the same time

### ISSUE THREAD

**Issue by Chrischuck** [spencermountain/spacetime#164]
The issue is pretty self explanatory once you see the code:

```
    const s = '2019-09-28T07:30:00.000-07:00'
    const s1 = spacetime(s).goto('America/Los_Angeles');
    const s2 = s1.goto('Pacific/Kiritimati').format('iso');
    const s3 = spacetime(s2).goto('America/Los_Angeles').format('iso')

    console.log(s) // '2019-09-28T07:30:00.000-07:00'
    console.log(s1.format('iso')) // '2019-09-28T07:30:00.000-07:00'
    console.log(s2) // 2019-09-29T04:30:00.000+14:00
    console.log(s3) // 2019-09-29T04:30:00.000-07:00
```

This works as expected if I switch to a timezone like Hawaii and back, but when timezones bring the date to another day I get this behavior. 

I'm probably missing something or not understanding something about timezones here

Thanks for the help!

**Comment by spencermountain** [comment]
hi @Chrischuck this is a good issue. I think i know what is happening.

it looks like it's actually an issue with the iso format, for a +14h offset, like `Kiritimati`. I didn't realize that offset existed.

For now, this works, if you wanted to just pass the spacetime object in...
```js
const s1 = spacetime('2019-09-28T07:30:00.000-07:00').goto('America/Los_Angeles')
// (7:30am in la on 28th)

const s2 = s1.goto('Pacific/Kiritimati')
// (4:30am next day in pacific)

// input spacetime object
const s3 = spacetime(s2).goto('America/Los_Angeles')
// back to 7:30am in la on 28th
```

i'll look at fixing it today

**Comment by spencermountain** [comment]
fixed in `6.1.0`
cheers
